spec/unit/history_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe Mongoid::History do
+  describe '.store' do
+    it 'should return RequestStore if requested' do
```

Comment:
To be picky, this is not what the spec tests in the non-RequestStore case. 

Create `spec/unit/store/request_store_spec.rb` and `spec/unit/store/default_store_spec.rb` and condition the entire spec on the presence of one or the other via the RSpec `if: ...`.